.github/workflows/publish-on-comment.yaml

```diff
@@ -0,0 +1,78 @@
+name: publish-on-comment
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  check-comment:
+    name: Check comment for /publish
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request }}
+    outputs:
+      trigger-comment: ${{ steps.check.outputs.triggered }}
+    steps:
+      - uses: khan/pull-request-comment-trigger@v1.1.0
+        id: check
+        with:
+          trigger: "/publish"
+          prefix_only: "true"
+          reaction: "rocket"
+
+  get-affected-plugins:
+    name: Get affected plugins
+    needs: check-comment
+    if: ${{ needs.check-comment.outputs.trigger-comment == 'true' }}
+    uses: ./.github/workflows/nx.template.yaml
+    with:
+      nx-head: ${{ github.head_ref && format('refs/pull/{0}/merge', github.event.number) || github.ref_name  }}
+      nx-base: ${{ github.base_ref || inputs.nx-base || github.event.repository.default_branch }}
+
+  get_last_commit:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request }}
+    outputs:
+      last_commit_sha: ${{ steps.last_commit.outputs.sha }}
+    steps:
+      - name: Get last commit SHA of the PR
+        id: last_commit
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const pr = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number
+            });
```

Comment:
doesn't this return an array? 
https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28

Reply:
Maybe it could be a better idea to extract the short sha directly here?